Cargo.toml

```diff
@@ -12,19 +12,20 @@ readme = "README.md"
 include = ["src/**/*", "README.md", "CHANGELOG.md"]
 
 [dependencies]
-mongodb = { version = "2", default-features = false }
+mongodb = { version = "3", default-features = true }
 serde = { version = "1", features = ["derive"] }
 futures-core = "0.3"
 futures-util = "0.3"
 async-trait = "0.1"
+#tokio = { version = "1", features = ["full"] }
 
 [dev-dependencies]
 tokio = "1.14.0"
 pretty_assertions = "1.0.0"
-chrono = { version = "0.4.19", features = [ "serde" ] }
+chrono = { version = "0.4.38", features = [ "serde" ] }
 
-[features]
-default = ["tokio-runtime"]
-tokio-runtime = ["mongodb/tokio-runtime"]
-async-std-runtime = ["mongodb/async-std-runtime"]
-chrono-0_4 = ["mongodb/bson-chrono-0_4"]
\ No newline at end of file
+#[features]
+#default = ["tokio-runtime"]
+#tokio-runtime = ["mongodb/tokio-runtime"]
+#async-std-runtime = ["mongodb/async-std-runtime"]
+#chrono-0_4 = ["mongodb/bson-chrono-0_4"]
```

Comment:
suggestion: Can you remove this completely if it’s not required anymore? I prefer deleting the lines instead of keeping them commented. Could you do that at the other places too?

Reply:
I removed those commented lines  completely from `Cargo.toml`